ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/index.tsx

```diff
@@ -95,22 +84,6 @@ const RunComponent = () => {
     runId: ortRun.id,
   });
 
-  if (ruleViolationsIsPending) {
-    return <LoadingIndicator />;
-  }
-
-  if (ruleViolationsIsError) {
-    toast.error('Unable to load data', {
-      description: <ToastError error={ruleViolationsError} />,
-      duration: Infinity,
-      cancel: {
-        label: 'Dismiss',
-        onClick: () => {},
-      },
-    });
-    return;
-  }
-
```

Comment:
Ah you do it in this commit. As they're in the same PR I'd move this to the initial implementation.

Reply:
If that's not your absolute requirement, I'd leave it as it is, as I don't want to spend any extra time struggling with GB with this, as it's not letting me amend.

Reply:
Generally the project values good commit history, personally I don't mind. @sschuberth as the resident commit bouncer, what do you say?

Reply:
I believe @Etsija has already addressed it, or? So no more need for me to look into it, correct?

Reply:
Sorry, I already removed my comment, as I found the solution fast, and didn't know you were scanning the PR while I quickly did it.